awscli/customizations/wizard/factory.py

```diff
@@ -31,3 +32,18 @@ def create_default_wizard_runner(session):
     )
     runner = core.Runner(planner, executor)
     return runner
+
+
+def create_default_wizard_v2_runner(session):
+    return WizardRunnerV2(session=session)
+
+
+class WizardRunnerV2(object):
+    def __init__(self, session):
+        self._session = session
+
+    def run(self, loaded):
+        """Run a single wizard given the contents as a string."""
+        app = create_wizard_app(loaded, self._session)
+        app.run()
```

Comment:
So in terms of refactoring the current v2 wizards, I was thinking, we:
1. Move the `create_wizard_app` to this module in order to maintain consistency with how things are declared in the `create_default_wizard_v1_runner`
2. Rename `WizardRunnerV2` to `WizardAppRunner` and move that class to `app.py`. Also update its `__init__()` to just accept a session and wizard app factory (e.g. the new signature would be something like:
```python
class WizardAppRunner:
   def __init__(self, session, app_factory):
        ...
```
3. In the `create_default_wizard_v2_runner`, pass the `create_wizard_app` function as the `app_factory` to the `WizardAppRunner` 
Thoughts? I think the main thing that I'm trying to get us to fix is to get all of the wiring logic that I put in `app.py` module to this factory class and do it in a way where the interface makes sense.